### DISCUSSION TITLE
Ctrl+W stopped work @ new version of PhotoDemon

### DISCUSSION THREAD

**Discussion by mesmariusz** [tannerhelland/PhotoDemon#448] [General]
Hi. 

Is there / somewhere a list of remapped key shortcuts between old and new version ?

E. g. Shortcut for Ctrl+W ( White balance ) stopped work @ new version of PhotoDemon.

Thank you for your support.

**Comment by tannerhelland** [comment]
Hello @mesmariusz .  I apologize that it has taken me so long to respond to your question!

In the last release, efforts were made to align PhotoDemon's key shortcuts ("hotkeys") with standard shortcuts in other photo editors.  `Ctrl+W` was remapped to `Close Image/Document`, which is the standard hotkey in photo editors (like Photoshop) and general Windows software (like Microsoft Word).  I sincerely apologize that this broke your existing workflow.

In the next release, I hope to make **all** shortcuts user-editable, so you can set whatever shortcuts you like for any function.  

In the meantime, do you have a suggested replacement hotkey for white balance?  I would be happy to add a temporary hotkey to nightly builds while I work on making all hotkeys editable.

**Reply by mesmariusz** [reply]
Hi @tannerhelland and thank you for your answer :-). 

Because of that change, I was able to find and realize, that this is possible to set permanently an other gamma ofset in the software for my  AIO scanner (Samsung Xpress C460FW). This way I solved my issue regarding a Little bit too bright images after scanning. Previous option it was to use White balance. Now I have this correction implemwnted  "by default".

But anyway for sure from time to time I will use the white balance. So I need to find some other shortcut ;-). I will think about it :-)

**Comment by hi5** [comment]
As a general remark: 

perhaps allow for chording keys, so not just <kbd>ctrl</kbd>+<kbd>k</kbd> but allow <kbd>ctrl</kbd>+<kbd>k</kbd> followed by another key: <kbd>ctrl</kbd>+<kbd>k</kbd> (release both keys) followed quickly by <kbd>w</kbd> - if it takes 'too long' nothing happens (usually 400-500ms, user config.).

This will allow for a lot of flexibility (and much more shortcut options)
